assets/css/base/gutenberg-blocks.scss

```
@@ -680,3 +680,11 @@
 		}
 	}
 }
+
+// Top rated products
```

Actually that affects all product grid blocks (Top rated, Best sellers, Hand-picked, etc.), I would either remove this comment line or change it for something like:
```suggestion
// Product grid blocks
```

---

Make sense, I first only applied to top-rated but then changed to grid products and didn’t update the comment